src/sentry/static/sentry/app/views/stream/stream.jsx

```diff
@@ -637,6 +638,14 @@ const Stream = createReactClass({
   },
 
   renderGroupNodes(ids, statsPeriod) {
+    // Restrict this guide to only show on a project with less than two issues
+    // because anchors light up all issues on the page
+    let userDateJoined = new Date(ConfigStore.get('user').dateJoined);
+    let dateCutoff = new Date();
+    dateCutoff.setDate(dateCutoff.getDate() - 30);
```

Comment:
Doesn't `getDate()` return the day of month as a number?  Why do we want that to be the case?

Reply:
Yes- which I need to subtract 30 days from and pass to `setDate` to update `dateCutoff` so that we only show it to people who joined less than 30days ago. I updated the comment to make it a little bit clearer. Does that make sense?

Reply:
ah, cool